docs/preset-env.md

```diff
@@ -229,40 +229,45 @@ This option is useful for "blacklisting" a transform like `@babel/plugin-transfo
 
 `"usage"` | `"entry"` | `false`, defaults to `false`.
 
-This option configures how `@babel/preset-env` handles polyfills.
+This option configures how `@babel/preset-env` imports polyfills and runtime support into your generated code.
 
-When either the `usage` or `entry` options are used, `@babel-preset-env` will add direct references to `core-js` modules as bare imports (or requires). This means `core-js` will be resolved relative to the file itself and needs to be accessible.
+Specifically, this option supports the following external packages:
 
-Since `@babel/polyfill` was deprecated in 7.4.0, we recommend directly adding `core-js` and setting the version via the [`corejs`](#corejs) option.
+- `core-js`: For standard JS built-ins
+- `regenerator-runtime`: For generator and `async`/`await` support
```

Comment:
Note that this isn't needed for `async`/`await` if your browser supports generators.

Reply:
Can you also mention this here?

Reply:
Isn't that assumed for this whole section, since `preset-env`'s entire purpose is to fill in for browsers' shortcomings.

i.e. `core-js` isn't needed if your browser supports all JS built-ins that it brings

Reply:
My point was that `regenerator-runtime` isn't always needed by browsers that don't support `async`/`await`, while this sentence implies the opposite.

`regenerator-runtime` is only needed for browsers that don't support generators.

Reply:
I see what you mean, but don't know exactly what text would make sense.

Do you have a suggestion?

Reply:
Maybe this?

```suggestion
- `regenerator-runtime`: For generators and `async`/`await` in browsers without generators support
```

Reply:
Done. Changed up your suggestion a little. I don't think everyone knows that `async`/`await` is built on generators, so I just grouped them together so it would be clear for everyone